wordpress-mobile/WordPress-FluxC-Android
# Block Editor Settings: Adds endpoint information to the editor theme change event

### Description
Collects the following information to be used for [analytics purposes](https://github.com/wordpress-mobile/WordPress-Android/pull/15009)
* [endpoint information to the editor theme change event](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2051/commits/5e176c5d847fda847e111957cf2b475d761c1fac)
* [full site editing flag to the editor theme payload](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2051/commits/7cc25c7e28bab21c777e15783716fab4b3f1ec21)

#### Depends on: https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2049

### To test
Check [Android PR](https://github.com/wordpress-mobile/WordPress-Android/pull/15009) to verify the collected data via the analytics event

📓  Note that the Jetpack side is still dependent on some backend changes so this PR is still a draft.
